sections/footer.liquid

```diff
@@ -235,7 +248,12 @@
     {%- endunless -%}
   {%- endif -%}
 
-  <div class="footer__content-bottom">
+  <div
+    class="footer__content-bottom{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}"
+    {% if settings.animations_reveal_on_scroll %}
+      data-cascade
+    {% endif %}
+  >
```

Comment:
Seem to works as expected in my tests 👍 

The one thing I wonder though is if it should be applied to the footer 🤔 

I personally don't feel like it adds much, and just like the header section, I prefer it not to animate. I can see how it could be odd that some do and some don't but since it's something that comes up on all pages, I prefer the impression that it's the content in the "middle" that needs to be loaded in. 

Curious to hear yours and other's thoughts on it.

Reply:
Yeah, @melissaperreault raised this earlier too. I went back and forth on it myself, but do I think it makes sense to animate the footer. 

The detail that convinced me most is the columned content: We're animating columned content in sequentially everywhere else it appears on the storefront, so it feels like a letdown if you get to the bottom and all of a sudden you get some columned content that does not animate anymore. Although the footer can be a little more utilitarian than the rest of the page, I think animating it helps maintain the continuity of the animation.